CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # FEniCS-preCICE adapter changelog
 
+## 1.3.0
+
+* Adding functionality for 3D cases with PointSource objects at coupling boundaries. See PRs [#133](https://github.com/precice/fenics-adapter/pull/133), [#146](https://github.com/precice/fenics-adapter/pull/146) and [#147](https://github.com/precice/fenics-adapter/pull/147)
```

Comment:
Do you also want to provide a link to the example in the tutorials here? I guess it might be useful, but maybe this is also more appropriate for the release notes.

Reply:
Please also summarize and mention https://github.com/precice/fenics-adapter/pull/150/commits/9dc6b4ebff5d8add2f1a9257c8e791ceb183d762.

Reply:
Summarized the config issue. We can link to the tutorial case in the release notes. Here it seems a bit excessive